packaging/nnstreamer.spec

```diff
@@ -311,6 +311,9 @@ BuildRequires:	pkgconfig(orc-0.4)
 BuildRequires:	flex
 BuildRequires:	bison
 
+# Mosquitto MQTT broker for unit testing
+BuildRequires:  mosquitto
```

Comment:
line 304 if mosquitto is used only for the unittest

Reply:
moved.